src/files.ts

```diff
@@ -122,7 +122,7 @@ export function getProjectFiles(rootDir: string = path.join('.', '/'), callback:
           // Preserves subdirectory names in rootDir
           // (rootDir/foo/Code.js becomes foo/Code.js)
           let formattedName = nameWithoutExt;
-          if (rootDir) {
+          if (rootDir && rootDir !== './' /* Extra check just in case so filename not mangled */) {
```

Comment:
Is this check really necessary?

Reply:
Not any more 🙂. Bug introduced by [PR #292](https://github.com/google/clasp/pull/292) ([9297b98](https://github.com/google/clasp/commit/9297b98df6c2c11033319b4e440a1fef0ebc84ee)). I think this is now superseded by [PR #323](https://github.com/google/clasp/pull/323).